src/component/registry/types.ts

```diff
@@ -161,9 +161,11 @@ export type ShapeStyleUpdate = EdgeStyleUpdate & { fill?: Fill };
  */
 export type Stroke = StyleWithOpacity & {
   /**
-   * Possible values are all HTML color names or HEX codes, as well as special keywords such as `swimlane`, `inherit` to use the color code of a related BPMN element or `none` for no color.
+   * Possible values are all HTML color names or HEX codes, as well as special keywords such as:
+   * - `default` to use the color defined in the BPMN element default style
+   * - `none` for no color
    */
-  color?: 'inherit' | 'none' | 'swimlane' | string;
```

Comment:
**question:** 

For my information, why did some values, available on `mxgraph`, has been removed ?

Reply:
this is explained in the description of the PR